validator/client/src/main/java/tech/pegasys/teku/validator/client/duties/AttestationProductionDuty.java

```diff
@@ -111,21 +112,23 @@ public String describe() {
                     }));
   }
 
-  private SafeFuture<Void> signAttestationsForCommittee(
+  private SafeFuture<DutyResult> signAttestationsForCommittee(
       final ForkInfo forkInfo, final Committee validators, final Attestation attestation) {
-    return validators.forEach(
-        validator -> signAttestationForValidator(forkInfo, attestation, validator));
+    return DutyResult.combine(
+        validators.forEach(
+            validator -> signAttestationForValidator(forkInfo, attestation, validator)));
   }
 
-  private SafeFuture<Void> signAttestationForValidator(
+  private SafeFuture<DutyResult> signAttestationForValidator(
       final ForkInfo forkInfo,
       final Attestation attestation,
       final ValidatorWithCommitteePosition validator) {
     return validator
         .getSigner()
         .signAttestationData(attestation.getData(), forkInfo)
         .thenApply(signature -> createSignedAttestation(attestation, validator, signature))
-        .thenAccept(validatorApiChannel::sendSignedAttestation);
+        .thenAccept(validatorApiChannel::sendSignedAttestation)
+        .thenApply(__ -> DutyResult.success(attestation.getData().getBeacon_block_root()));
```

Comment:
Do we not want the attestation root here?

Reply:
The logs (when they used to work) have always shown the block the attestation is attesting to.  That' actually a lot more useful because what you're most interested in is whether the attestation will wind up being on the correct chain and thus get rewarded.